dev-python/snakemake/snakemake-8.28.0.ebuild

```
@@ -0,0 +1,144 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..12} )
+DISTUTILS_USE_PEP517=setuptools
+
+inherit distutils-r1
+
+DESCRIPTION="Make-like task language"
+HOMEPAGE="https://snakemake.readthedocs.io"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-python/appdirs[${PYTHON_USEDEP}]
+	dev-python/immutables[${PYTHON_USEDEP}]
+	dev-python/configargparse[${PYTHON_USEDEP}]
+	>=dev-python/connection_pool-0.0.3[${PYTHON_USEDEP}]
+	dev-python/datrie[${PYTHON_USEDEP}]
+	dev-python/docutils[${PYTHON_USEDEP}]
+	dev-python/gitpython[${PYTHON_USEDEP}]
+	dev-python/humanfriendly[${PYTHON_USEDEP}]
+	>=dev-python/jinja-3.0[${PYTHON_USEDEP}]
+	dev-python/jsonschema[${PYTHON_USEDEP}]
+	dev-python/nbformat[${PYTHON_USEDEP}]
+	dev-python/packaging[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.8.1[${PYTHON_USEDEP}]
+	dev-python/reretry[${PYTHON_USEDEP}]
+	>=dev-python/smart-open-4.0[${PYTHON_USEDEP}]
+	>=dev-python/snakemake-interface-common-1.17.0[${PYTHON_USEDEP}]
+	>=dev-python/snakemake-interface-executor-plugins-9.3.2[${PYTHON_USEDEP}]
+	>=dev-python/snakemake-interface-storage-plugins-3.2.3[${PYTHON_USEDEP}]
+	>=dev-python/snakemake-interface-report-plugins-1.1.0[${PYTHON_USEDEP}]
+	dev-python/stopit[${PYTHON_USEDEP}]
+	dev-python/tabulate[${PYTHON_USEDEP}]
+	dev-python/throttler[${PYTHON_USEDEP}]
+	dev-python/wrapt[${PYTHON_USEDEP}]
+	>=dev-python/yte-1.5.5[${PYTHON_USEDEP}]
+	>=dev-python/dpath-2.1.6[${PYTHON_USEDEP}]
+	>=dev-python/conda-inject-1.3.1[${PYTHON_USEDEP}]
+	>=sci-mathematics/pulp-2.3.1[${PYTHON_USEDEP}]
+"
+DEPEND="
+	${RDEPEND}
+	test? (
+		dev-python/pytest-mock
```

```suggestion
		dev-python/pytest-mock[${PYTHON_USEDEP}]
```

And this should be BDEPEND

---

Fixed this